src/QueryCollection.js

```diff
@@ -47,6 +48,7 @@ export default function(match, selector, init){
 
       this._isQueryCollection = true
       this.length = elements.length;
+      this.renderer = ReactTestUtils.createRenderer()
```

Comment:
So this isn't actually the place for this, since it will create a renderer for InstanceCollections as well. I would just create a new renderer in the `shallowRender()` (like before) method and attach it to the instance then, so it can be reused on subsequent renders